start.go

```diff
@@ -85,12 +85,17 @@ command(s) that get executed on start, edit the args parameter of the spec. See
 	},
 }
 
+func init() {
+	if len(os.Args) > 1 && os.Args[1] == "init" {
+		runtime.GOMAXPROCS(1)
```

Comment:
Not that it matters since we don't have other goroutines, but is the call to GOMAXPROCS(1) required?